pkg/blockstorage/azure/client.go

```diff
@@ -82,12 +97,14 @@ func NewClient(ctx context.Context, config map[string]string) (*Client, error) {
 		return nil, err
 	}
 	cred := authenticator.GetAuthorizer()
-	computeClientFactory, err = armcompute.NewClientFactory(subscriptionID, cred, nil)
+	opts := &arm.ClientOptions{ClientOptions: policy.ClientOptions{}}
+	opts.ClientOptions.Cloud = GetCloudConfig(config[blockstorage.AzureCloudEnvironmentID])
```

Comment:
I believe this code makes any difference because it's not possible to pass `blockstorage.AzureCloudEnvironmentID` from the https://github.com/kanisterio/kanister/blob/master/pkg/function/create_volume_snapshot.go#L298

Reply:
ok seems it's dead code / too much effort to change. Closing the PR